### ISSUE TITLE
iOS problem

### ISSUE THREAD

**Issue by PawelLegowski** [mauron85/cordova-plugin-background-geolocation#97]

**Comment by PawelLegowski** [comment]
Hello!
I have problem with location on iOS. It turns out to have poor range and not update over time. It works fine on Android, but iOS is causing troubles.

I configure plugin with:
 window.plugins.backgroundGeoLocation.configure(callbackFn, failureFn, {
                desiredAccuracy: 10,
                distanceFilter: 20,
                stationaryRadius: 30,
                activityType: 'Fitness',
                debug: true,
                stopOnTerminate: true
   });

I was trying to use window.plugins.backgroundGeoLocation.changePace function, or remove stationaryRadius param, but it did not solve my problem, does anyone managed to implement GPS constant tracking on IOS?

**Comment by mauron85** [comment]
Yes, version 2.0.0 is works also in fg and bg. However in bg it's only using significant changes and region monitoring, which is not as same accurate as in fg mode, but save battery a lot. Anyway I will reverse that decision and add optional parameter that forces plugin to be as same as accurate as in fg. Wait for version 2.1.0.

**Comment by PawelLegowski** [comment]
Thank You for answer!
Currently I am working on GPS game (you have to come close to point on map to progress game), and I need to have precise location whole time (i am sending local notification while in BG), so probably I should wait for update, because switching to foreground manually doesn't work (switches back to background automatically as stated in doc "iOS foreground mode witch automatic background mode switch")?

**Comment by mauron85** [comment]
Please try version 2.1.0. It is in next branch. And set `option.saveBatteryOnBackground: false`

 You can install it from npm:
```
cordova plugin add cordova-plugin-mauron85-background-geolocation@next
```

**Comment by mauron85** [comment]
Also don't forget to also set stopOnTerminate: false if you want to monitor also in the background
